shared/settings/nav/index.desktop.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React, {Component} from 'react'
+import {globalStyles, globalColors, globalMargins} from '../../styles'
+import {Box, Badge, ClickableBox, Text} from '../../common-adapters'
+
+import type {Props, BannerType, SettingsItem as SettingsItemType} from './index'
+
+function Banner ({element, type}: {element: React$Element<*>, type: BannerType}) {
+  return (
+    <Box style={{...commonBannerStyle, ...variantBannerStyle[type]}}>
```

Comment:
(Oh yeah, this is a :sheep:! I got confused by the new reviewing UI and forgot to go back and label it as such.)